processor/src/test/java/org/mapstruct/ap/test/nestedbeans/mixed/AutomappingAndNestedTest.java

```diff
@@ -200,7 +204,7 @@ public void shouldAutomapAndHandleSourceAndTargetPropertyNestingAndExpresion() {
 
         assertThat( target.getQuality() ).isNotNull();
         assertThat( target.getQuality().getReport() ).isNotNull();
-        assertThat( target.getQuality().getReport().getVerdict() ).isEqualTo( "PASSED" );
```

Comment:
You are right. Maybe if we don't have anything we just need to leave it and let MapStruct do name based mapping here